src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+.reset {
+  margin: 0;
+  background-color: #e5e5e5;
```

Comment:
this is not part of header so not required to have` background-color`

Reply:
I just use it same as in Mock. Because in mock header bgc: #fff and body bgc: #E5E5E5. and .reset is body class